src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -1635,7 +1635,8 @@ GetDependentSequencesWithRelation(Oid relationId, List **seqInfoList,
 			attrdefResult = lappend_oid(attrdefResult, deprec->objid);
 			attrdefAttnumResult = lappend_int(attrdefAttnumResult, deprec->refobjsubid);
 		}
-		else if (deprec->deptype == DEPENDENCY_AUTO &&
+		else if ((deprec->deptype == DEPENDENCY_AUTO || deprec->deptype ==
```

Comment:
This line causes https://github.com/citusdata/citus/pull/6591#issuecomment-1380385403